testinfra/modules/file.py

```diff
@@ -228,9 +230,7 @@ def md5sum(self):
 
     @property
     def sha256sum(self):
-        return self.check_output(
-            "sha256sum %s | cut -d ' ' -f 1", self.path)
-
```

Comment:
@daenney thanks ! But for linux I prefer keep sha256sum which is part of the GNU coreutils. So let's keep shasum only for OSX.